lib/metrics.js

```diff
@@ -125,6 +126,21 @@ module.exports = {
     }))
   },
 
+  watchLoadingOfUserDefinedKeyBindings () {
```

Comment:
this is a very descriptive method name.

![bitmoji](https://render.bitstrips.com/v2/cpanel/1129d23f-9163-4de7-a1d6-443d34008eaf-f14a1e1d-64bc-41f2-8006-01b6c63adbd8-v1.png?transparent=1&palette=1&width=246)